src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js

```diff
@@ -28,6 +28,33 @@ function isClusterAvailable(cluster) {
   return ready;
 }
 
+function getClusterLastCheckedIn(cluster) {
+  let ready = false;
+  if (
+    cluster?.spec.clusterData &&
+    cluster?.spec.clusterData.cluster_status &&
+    cluster?.spec.clusterData.cluster_status.conditions &&
+    cluster?.spec.clusterData.cluster_status.conditions.length > 0
```

Comment:
I guess we can do `cluster?.spec?.clusterData?.cluster_status?.conditions?.length` here instead.

Reply:
makes sense, changed this.